### DISCUSSION TITLE
An adult female bear is called a what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#411] [Galaxy Trivia] [answered]
**Hint:** 🪚

**Comment by peterbaccaro** [comment]
An adult female bears is called a **sow**.

**Comment by soto-sergio** [comment, marked as answer]
Sows are adult female bears capable of reproducing. Fun fact: Adult male bears are called boars. 🐻 🐗

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/soto-sergio?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
